src/timepicker/stories/index.css

```diff
@@ -27,11 +54,23 @@
   border: 1px solid #838bfa;
 }
 
-.timepicker__column--value {
+.timepicker__content .timepicker__column--value {
   padding: 5px;
 }
 
-[data-selected="true"] {
+..timepicker__content [data-selected="true"] {
   background-color: #3c49fd;
   color: white;
 }
+
+.timepicker__content ::-webkit-scrollbar {
+  width: 5px;
+}
+
+.timepicker__content ::-webkit-scrollbar-track {
+  background-color: #dde0ff;
+}
+
+.timepicker__content ::-webkit-scrollbar-thumb {
+  background-color: #9097ff;
+}
```

Comment:
Add this CSS

```css

.timepicker__column--value[data-selected="true"] {
  background-color: #1e65fd;
  color: white;
}
```
it will show the selected time value..
![image](https://user-images.githubusercontent.com/35374649/95427155-b7cd5980-0964-11eb-97ec-e9ea10e6b513.png)